as/thrift.js

```diff
@@ -45,6 +45,7 @@ function TChannelAsThrift(opts) {
         'must pass source as an argument');
 
     self.thriftSource = opts.source;
+    self.thriftFileName = opts.thriftFileName || 'service';
```

Comment:
Not sure like the idea of having a dot in the map key. Plus, it should always be ".thrift'. Will make the change if got a '+1' on 'service.thrift'. :)